web_tree_many2one_clickable/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Better to use the png images, see the latest [template](https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)

Reply:
stupid issue https://github.com/OCA/maintainer-tools/commit/52f41366ab39ea90e66f3e3a2b5cd7691b4219df#diff-cd6819c720069eb20ba0915c04fdebe1 :smile: 
SVG are way better and lighter.... /END_OF_RANT :stuck_out_tongue:

Reply:
I prefer the images that are visible ;)

Reply:
hehe, was not complaining about that, I was complaining about the issue that caused it :wink: